include/capstone/x86.h

```diff
@@ -300,23 +308,6 @@ typedef struct cs_x86 {
 	// Address size, which can be overrided with above prefix[5].
 	uint8_t addr_size;
 
-	// ModR/M byte
-	uint8_t modrm;
```

Comment:
this patch moves modrm, sib, disp, sib_index, sib_scale, sib_base, etc into new cs_x86_encoding struct, thus breaks compatibility, and breaks it hard. how about keeping these fields in the old place, and cs_x86_encoding only contains new data xxx_offset, so old code still works?